File: Solutions/0162/0162.cpp
class Solution {
public:
    int findPeakElement(vector<int>& nums) {
        int l=0, r=nums.size()-1;
        while (l<r){
            int m = (l+r)/2;
            nums[m] < nums[m+1]? l=m+1 : r=m;
        }
        return l;
    }
};

File: Solutions/0162/0162.py
# -*- coding:utf-8 -*-
# Author: washing
# DateTime: 2021/9/15 17:38
# File: 0162.py
# Desc: 
class Solution:
    def findPeakElement(self, nums: List[int]) -> int:
        return nums.index(max(nums))
